tests/test_polaraveraging.py

```diff
@@ -0,0 +1,225 @@
+"""
```

Comment:
No tests for

- [ ] '_astropy_to_CCL_cosmo_object', (I think this should be removed anyways)
- [ ]  'make_shear_profile',
- [ ]  'plot_profiles', (I think this should be moved to a plotting submodule)